osquery/tables/yara/yara.cpp

```diff
@@ -39,71 +44,306 @@ FLAG(bool,
      true,
      "Call malloc_trim() after YARA scans (linux)");
 #endif
+
 FLAG(uint32,
      yara_delay,
      50,
      "Time in ms to sleep after scan of each file (default 50) to reduce "
      "memory spikes");
 
+HIDDEN_FLAG(bool,
+            enable_yara_sigrule,
+            false,
+            "Enable yara table extension to pass sigrule with query ");
+
+HIDDEN_FLAG(bool,
+            enable_yara_sigurl,
+            false,
+            "Enable yara table extension to pass sigurl with query ");
+
+HIDDEN_FLAG(bool,
+            disable_yara_string_private,
```

Comment:
I'd prefer a flag that follows the "enable" convention. As in, `enable_yara_string_results`. Thinking as the user, I want to enable something when I set this flag: the returning of matched strings. The way it is now seems confusing, and will require documentation like "enable the disable_yara_string_private flag to disable the hiding" instead of "enable the enable flag to show the strings".